pkg/services/store/kind/dashboard/summary.go

```diff
@@ -0,0 +1,106 @@
+package dashboard
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strconv"
+	"sync"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/services/store"
+)
+
+func GetObjectKindInfo() models.ObjectKindInfo {
+	return models.ObjectKindInfo{
+		ID:          models.StandardKindDashboard,
+		Name:        "Dashboard",
+		Description: "Define a grafana dashboard layout",
+	}
+}
+
+func NewDashboardSummary(sql *sqlstore.SQLStore) models.ObjectSummaryBuilder {
+	cache := make(map[int64]models.ObjectSummaryBuilder)
+	mutex := sync.Mutex{}
+
+	return func(ctx context.Context, uid string, body []byte) (*models.ObjectSummary, []byte, error) {
+		user := store.UserFromContext(ctx)
```

Comment:
I am not sure about this - building the entity summary based on any data coming from a user feels off. The summary should be the same regardless of who initiates the request; it's the responsibility of a layer above to filter/hide certain properties a user may have no access to.

Reply:
it only gets the orgID -- and that should/will soon be passed in with a "GRN" rather than the uid